src/applications/bmqstoragetool/m_bmqstoragetool_searchresult.cpp

```diff
@@ -81,7 +81,7 @@ void printJournalFileMeta(bsl::ostream&              ostream,
     fields.push_back("SyncPoint DataFileOffset (DWORDS)");
     fields.push_back("SyncPoint QlistFileOffset (WORDS)");
 
-    bmqu::AlignedPrinter printer(ostream, &fields);
+    bmqu::AlignedPrinter printer(ostream, fields);
```

Comment:
Also need to change type, and do the same in other places in this file:
`bsl::vector<const char*> fields` -> `bsl::vector<bsl::string> fields`